tests/test_manager.py

```diff
@@ -63,30 +63,31 @@ def test_init_app(self):
         :class:`flask.ext.restless.APIManager` object.
 
         """
-        manager = APIManager()
-        manager.init_app(self.flaskapp, session=self.session)
-        manager.create_api(self.Person, app=self.flaskapp)
+        manager = APIManager(session=self.session)
+        manager.create_api(self.Person)
+        manager.init_app(self.flaskapp)
```

Comment:
Agreed, these tests should be redesigned.
It actually may be possible to support this feature in my current fork by making the app parameter in create_api mandatory and register the blueprints to the app parameter immediately after creation. In which case create_api would work as a sort of init_app with passed in parameters and no memory about the blueprint created such that a subsequent call to app_init would not apply the blueprints created via create_api to the app later initialized via init_app.